AirCasting/Models/ThresholdsValue.swift

```diff
@@ -0,0 +1,33 @@
+// Created by Lunar on 19/11/2021.
+//
+
+import Foundation
+
+public struct ThresholdsValue {
+    let veryLow: Int32
+    let low: Int32
+    let medium: Int32
+    let high: Int32
+    let veryHigh: Int32
+    
+    var toArray: [Int32] {
+        [veryLow, low, medium, high, veryHigh]
+    }
+    
+    init(array: [Int32]) {
+        assert(array.count == 5)
```

Comment:
Here we have to make an assumption that we will get an array with 5 elements. Probably it would be better to remove this initializer and split the array into appropriate values in the view model (as mentioned in another comment)